### PR TITLE
fix: settings dialog files path copy snackbar

### PR DESCRIPTION
**Pull request by Xazin**
Relates: #2056 

<details>
  <summary>Screen recording of how it looks</summary>

Screen Recording 2023-03-26 at 22 09 50

</details>

### CODE REVIEW

**Comment by Xazin** [comment]
Not sure what is going on with the Snackbar theme, but that should be fixed in another issue.

**Review by a-wallen — APPROVED** [review]
LGTM

**Comment by a-wallen** [comment]
@Xazin can you add an issue to fix the dialog theme?

**Comment by Xazin** [comment]
> @Xazin can you add an issue to fix the dialog theme and mention this issue in it?

Will do.

**Comment by a-wallen** [comment]
Also, please add a test at your convenience 😁

File: frontend/appflowy_flutter/packages/flowy_infra_ui/lib/src/flowy_overlay/flowy_dialog.dart [hunk 1]
```diff
@@ -25,14 +25,14 @@ class FlowyDialog extends StatelessWidget {
...
     return SimpleDialog(
+        contentPadding: EdgeInsets.zero,
         title: title,
         shape: shape ??
             RoundedRectangleBorder(borderRadius: BorderRadius.circular(8)),
         children: [
           Material(
             type: MaterialType.transparency,
             child: Container(
-              padding: padding,
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/packages/flowy_infra_ui/lib/src/flowy_overlay/flowy_dialog.dart:35]: Why do you delete this line?

Comment (Xazin) [reply frontend/appflowy_flutter/packages/flowy_infra_ui/lib/src/flowy_overlay/flowy_dialog.dart:35]: To not have the Snackbar have horizontal padding, but to fill the length of the dialog.

I've substituted it further down in the tree.

Comment (Xazin) [reply frontend/appflowy_flutter/packages/flowy_infra_ui/lib/src/flowy_overlay/flowy_dialog.dart:35]: I can also make it have the horizontal padding, and make the snackbar rounded, we need a design for snackbar anyway, so we can change it then. I wasn't able to find it at least.

**Pull request merged**